src/pages/guidelines/spacing.mdx

```diff
@@ -0,0 +1,126 @@
+
+When designing for IBM.com, the spacing scale and type tokens are used to ensure the information is well organized with consistency.
+
+<br />
+
+## Spacing token and scale
+
+### Spacing token
+
+The spacing scale can be applied to `margin` or `padding` properties, as well as
+to both vertical and horizontal edges. The token takes the place of the values
+normally assigned to `margin` and `padding`. The following are all approved ways
+to syntactically apply Carbon for IBM.com spacing tokens:
+
+```css
+margin: $spacing-03;
+margin: $spacing-03 $spacing-01;
+margin: $spacing-07 0 $spacing-04 0;
+margin-right: $spacing-05;
+padding: $spacing-04;
+padding: $spacing-05 $spacing-03;
+padding: $spacing-07 $spacing-04 0 $spacing-04;
+```
+<br />
+
+Below is the example of a regular image card on the left and how the spacing tokens are applied to it on the right.
+
+<Row>
+<Column colLg={8}>
+
+![Example of spacing tokens applied](../../images/guidelines/spacing/spacing-example-01.jpg)
+
+
+
+</Column>
+</Row>
+
+<br />
+
+### Spacing scale
+
+Use the spacing scale when building individual components. It includes small
+increments needed to create appropriate spatial relationships for detail-level
+designs. This scale is applied and used within all Carbon components.
+
+<div className="spacing-table">
+
+| Token         | rem   | px  | Example                                       |
+| ------------- | ----- | --- | --------------------------------------------- |
+| `$spacing-01` | 0.125 | 2   | ![spacing-3xs example](../../images/guidelines/spacing/spacing-01.svg) |
+| `$spacing-02` | 0.25  | 4   | ![spacing-2xs example](../../images/guidelines/spacing/spacing-02.svg) |
+| `$spacing-03` | 0.5   | 8   | ![spacing-xs example](../../images/guidelines/spacing/spacing-03.svg) |
+| `$spacing-04` | 0.75  | 12  | ![spacing-sm example](../../images/guidelines/spacing/spacing-04.svg) |
+| `$spacing-05` | 1     | 16  | ![spacing-md example](../../images/guidelines/spacing/spacing-05.svg) |
+| `$spacing-06` | 1.5   | 24  | ![spacing-lg example](../../images/guidelines/spacing/spacing-06.svg) |
+| `$spacing-07` | 2     | 32  | ![spacing-xl example](../../images/guidelines/spacing/spacing-07.svg) |
+| `$spacing-08` | 2.5   | 40  | ![spacing-2xl example](../../images/guidelines/spacing/spacing-08.svg) |
+| `$spacing-09` | 3     | 48  | ![spacing-3xl example](../../images/guidelines/spacing/spacing-09.svg) |
+| `$spacing-10` | 4     | 64  | ![spacing-4xl example](../../images/guidelines/spacing/spacing-10.svg) |
+| `$spacing-11` | 5     | 80  | ![spacing-5xl example](../../images/guidelines/spacing/spacing-11.svg) ||
+| `$spacing-12` | 6     | 96  | ![spacing-6xl example](../../images/guidelines/spacing/spacing-12.svg) |
+| `$spacing-13` | 10    | 160 | ![spacing-7xl example](../../images/guidelines/spacing/spacing-13.svg) |
```

Comment:
Here we go... Try this...

Need to delete the others, but keep one. I'd suggest a general name like just `spacing.svg` as I have here. Then just make sure the image is pointed to the correct path.

```suggestion
| Token         | rem   | px  | Example                                       |
| ------------- | ----- | --- | --------------------------------------------- |
| `$spacing-01` | 0.125 | 2   | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 01 example" style="width: 2px; height: 2px;" />     |
| `$spacing-02` | 0.25  | 4   | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 02 example" style="width: 4px; height: 4px;" />     |
| `$spacing-03` | 0.5   | 8   | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 03 example" style="width: 8px; height: 8px;" />     |
| `$spacing-04` | 0.75  | 12  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 04 example" style="width: 12px; height: 12px;" />   |
| `$spacing-05` | 1     | 16  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 05 example" style="width: 16px; height: 16px;" />   |
| `$spacing-06` | 1.5   | 24  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 06 example" style="width: 24px; height: 24px;" />   |
| `$spacing-07` | 2     | 32  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 07 example" style="width: 32px; height: 32px;" />   |
| `$spacing-08` | 2.5   | 40  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 08 example" style="width: 40px; height: 40px;" />   |
| `$spacing-09` | 3     | 48  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 09 example" style="width: 48px; height: 48px;" />   |
| `$spacing-10` | 4     | 64  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 10 example" style="width: 64px; height: 64px;" />   |
| `$spacing-11` | 5     | 80  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 11 example" style="width: 80px; height: 80px;" />   |
| `$spacing-12` | 6     | 96  | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 12 example" style="width: 96px; height: 96px;" />   |
| `$spacing-13` | 10    | 160 | <img src="../../images/guidelines/spacing/spacing.svg" alt="Spacing 13 example" style="width: 160px; height: 160px;" /> |
```

Looks like our website is out of date because the spacing package just has 12 tokens, and not 13. Otherwise, I'd suggest let's just dynamically pull the data from there so we never have to update this list again. 🤷‍♂️